src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNamedPipesTest/SqlNamedPipesTest.cs

```diff
@@ -40,26 +34,24 @@ public static void InvalidConnStringTest()
             SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.NpConnStr);
             builder.ConnectTimeout = 2;
 
-            string host = GetHostFromDataSource(builder.DataSource);
-
             // Using forward slashes
-            builder.DataSource = "np://" + host + "/pipe/sql/query";
```

Comment:
These tests should fail at the connection string parsing phase, so having trying to connect to a server is pointless. If one of these cases somehow gets through named pipes parsing, then the connection will time out when trying to connect to a nonexistent server. Just seeing a different error message in that case is sufficient to show up as a test failure.